src/views/Dashboard/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { DashboardLayout } from "./layout";
+import { useUserDepositAssetBalancesDaiWei } from "../../queries/userAssets";
+import { DepositAsset } from "../Deposit";
+
+function Dashboard() {
```

Comment:
FCs are preferred, and top-level type definitions are required. Also, prefer non-`default` exports.
```suggestion
export const Dashboard: React.FC<{}> = (_props) => {
```